core/src/test/scala/org/apache/spark/network/netty/NettyBlockTransferSecuritySuite.scala

```diff
@@ -112,6 +179,19 @@ class NettyBlockTransferSecuritySuite extends SparkFunSuite with MockitoSugar wi
     }
   }
 
+  test("security with aes encryption over ssl") {
```

Comment:
This is [not supported](https://github.com/apache/spark/blob/376de8a502fca6b46d7f21560a60024d643144ea/core/src/main/scala/org/apache/spark/SecurityManager.scala#L278) currently, drop this ?

Reply:
two things:

1. Once I merge the tests per your suggestion, this will run still.
2. I think it's nice to test that we do not consider this misconfiguration fatal (with that warning being logged) - what do you think?

Reply:
Agree, it is not a case we fail over. Let us keep it.